CHANGELOG.rst

```diff
@@ -5,6 +5,20 @@ The following is a summary of changes and improvements to
 :mod:`eulxml`.  New features in each version should be listed, with
 any necessary information about installation or upgrade notes.
 
+1.1
+-----
+
+* Closed issue: use xml catalog for loading schemas #18
```

Comment:
Change log entries should be more like a user story or high-level description of the feature/changes, not a blow-by-blow of specifics (which can be found in the commit history anyway).  The goal is for a user to be able to glance at the change log and see if the change affects them or their code that uses this library so they know if they need to make any updates (in the past, it's helped me track down a bug or weird interaction with other libraries).  I think this is so specific it won't be meaningful to anyone but us.

You can look at the other releases for examples, or I can help you figure out how to draft it.